library/init/meta/interactive.lean

```diff
@@ -6,16 +6,21 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean.parser
+-- note: not used directly, but by C++ when parsing tactic args
+import init.meta.uneval
+
+open lean
+open lean.parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : parser α) : Type := α
```

Comment:
I think we should add the `has_uneval` instance here as an extra argument, then we 1) don't need to do instance search while parsing, 2) get error messages for missing `has_uneval` instances already when defining interactive tactics (not just when using them).